### PR TITLE
N14: Recall Singularity

### PR DESCRIPTION
**Pull request by AngelOnFira**
#278 :tada:

### CODE REVIEW

**Review by Terkwood — APPROVED** [review]

File: content/posts/newsletter-014/index.md [hunk 1]
```diff
@@ -204,6 +204,28 @@ revamped netcode and the full core set of units.
 
 [ggez]: https://ggez.rs/
 
+### Recall Singularity
+
+![Demo of the basic ship collision](recall_singularity.png) _A ray-traced
+selection of modules_
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-014/index.md:210]:
```suggestion
![Demo of the basic ship collision](recall_singularity.png)
_A ray-traced selection of modules_
```

File: content/posts/newsletter-014/index.md [hunk 2]
```diff
@@ -204,6 +204,28 @@ revamped netcode and the full core set of units.
...
+
+The Recall Singularity is a game about designing autonomous factory ships and
+stations created by [Tom Leys][Tom Leys]. This month a new devlog was posted:
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-014/index.md:213]:
```suggestion
stations created by [Tom Leys]. This month a new devlog was posted:
```

File: content/posts/newsletter-014/index.md [hunk 3]
```diff
@@ -204,6 +204,28 @@ revamped netcode and the full core set of units.
...
+here][recall-s-sept-video].
+
+Summary:
+
+- Improving the robustness of the game core and networking
+- Preliminary factory features
+- Optimizing network encoding
+
+[Tom Leys]: https://twitter.com/RecallSingular1 [recall-s-sept-text]:
+https://medium.com/@recallsingularity/recall-singularity-in-sep-2020-e2f33a85fd7c
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-014/index.md:226]:
```suggestion
[Tom Leys]: https://twitter.com/RecallSingular1
[recall-s-sept-text]: https://medium.com/@recallsingularity/recall-singularity-in-sep-2020-e2f33a85fd7c
```

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
Content looks good, just a few issues with formatting and links.

**Comment by ozkriff** [comment]
🚧

**Review by ozkriff — APPROVED** [review]
:rocket: (optimized the image and made a few style/wording tweaks)

**Review by 17cupsofcoffee — APPROVED** [review]
🚀

**Pull request merged**